public/js/form_helpers.js

```diff
@@ -20,42 +20,64 @@ $(document).ready(function(){
         .on('focusout', function() {
             validation($(this));
         });
+    $('.mask-zip')
+        .on('focusout', function() {
+            validateZip($(this));
+        })
 });
 
 function initMask(){
     $('.mask-zip').mask('00000');
-    $('.mask-phone').mask('(000) 000-0000');
+    $('.mask-phone').mask('0000000000');
```

Comment:
Is there a reason this was changed from the `(###) ###-####` format to just numbers?

Reply:
The reason I did this was because sometimes the user doesn't provide an area code and the format looks kind of weird in the db. I could easily change it back though if we wanted to say that no area code defaults to the 845 area code. It's really up to you guys.